components/faucet/CTA.tsx

```diff
@@ -0,0 +1,49 @@
+import { Flex } from "@chakra-ui/react";
+import Link from "next/link";
+import type { FC } from "react";
+import { Button, Text } from "tw-components";
+
+const CTA: FC = () => {
+  return (
+    <Flex
+      bg="url(/assets/faucet/cta-bg.png) no-repeat center"
+      bgSize="cover"
+      flexDir="column"
+      gap="6"
+      h="226px"
```

Comment:
not loving the hard coded height here, can we use an `<AspectRatio>` to achieve the same goal? this hard-coded height means this banner will be huge on mobile